packages/twenty-front/src/modules/action-menu/components/ActionMenuBarEntry.tsx

```diff
@@ -41,6 +41,10 @@ export const ActionMenuBarEntry = ({ entry }: ActionMenuBarEntryProps) => {
     <StyledButton
       accent={entry.accent ?? 'default'}
       onClick={() => entry.onClick?.()}
+      data-testid={entry.label
```

Comment:
for example why do we need this one? can't we select based on entry.label directly (this will select the label but I guess it's still fine to trigger DOM events on parents)

Reply:
This one isn't necessary and can be located using text, I wanted to make life a bit easier but I'll refactor it, re-check my changes and delete attributes from places which can be located using text